fix: Adjusted documentation for report generation to use nf-aggregate (#5)
seqeralabs/fusion-benchmarking-scripts: 05_generate_report/README.md

### 1. Prerequisites

> - You have successfully completed Nextflow pipeline runs with both Fusion and plain S3.

Reviewer: ```suggestion
- You have successful executions of your Nextflow pipeline(s) with both Fusion and plain S3.
```

- You have access to the Seqera Platform workspace with the completed runs.

> - You have access to an AWS cost and usage report (CUR) in parquet format, containing cost information for your benchmarking runs with the resource labels we have set up with you.

Reviewer: ```suggestion
- You have access to an AWS cost and usage report (CUR) in parquet format, containing cost information for your benchmarking runs with the resource labels we have set up with you.
```
Is this going to only support for 1.0 or 2.0 as well? If we're only going to support one (for now), let's clarify which one here.

Author: I would try to get the CUR2.0 support added to the benchmark reports today and update the nf-aggregate benchmark container tag. That way we would be safe for both!

> - **Task Instance Usage**: This subsection shows the number of tasks that ran on different instance types during pipeline runs, allowing for quick comparisons of instance type usage between groups. Users can hover over the stacked bar plots to view the detailed distribution of instance types and can use the legend to highlight or hide specific instance types.

Author: Hover hints was something Harshil and I always planned but I couldn't yet figure out how to do this in Quarto. Hopefully Cursor can be helpful with this :rocket:!

> - **Task Instance Usage**: This subsection shows the number of tasks that ran on different instance types during pipeline runs, allowing for quick comparisons of instance type usage between groups. Users can hover over the stacked bar plots to view the detailed distribution of instance types and can use the legend to highlight or hide specific instance types.

Author: I have incorporated most of your suggestions here as bullet points under run overview.
